safe/library.py

```diff
@@ -15,11 +15,19 @@ class APIError(requests.HTTPError):
 
 class Reason(object):
     def __init__(self, reason):
-        self.name = reason.get('obj_name')
-        self.obj = reason['obj_type']
-        self.description = reason['description']
-        self.module = reason['module']
-        self.url = reason.get('url')
+        # if it's a string, just treat it as the error description
+        if isinstance(reason, basestring):
```

Comment:
Use `six.string_type` (https://pythonhosted.org/six/#six.string_types). I support python3.

I already pull in six as a dependency.